scripts/xenadmin-build.sh

```diff
@@ -225,14 +225,14 @@ msi_checksum=($msi_checksum_with_file_name)
 
 sha256sum ${OUTPUT_DIR}/${BRANDING_BRAND_CONSOLE_NO_SPACE}-source.zip > ${OUTPUT_DIR}/${BRANDING_BRAND_CONSOLE_NO_SPACE}-source.zip.checksum
 
-echo "INFO: Generating CHCUpdates.xml"
+echo "INFO: Generating XCUpdates.xml"
 
 # UPDATE_URL points at the updates XML, we need to point to the MSI
-msi_url="${UPDATES_URL/CHCUpdates.xml/$BRANDING_BRAND_CONSOLE_NO_SPACE.msi}"
+msi_url="${UPDATES_URL/XCUpdates.xml/$BRANDING_BRAND_CONSOLE_NO_SPACE.msi}"
 
 output_xml="<?xml version=\"1.0\" ?>
 <patchdata>
-    <chcversions>
+    <versions>
```

Comment:
The tag is also used in DownloadUpdatesXmlAction.cs.

Reply:
oops! well spotted 😅